Makefile

```diff
@@ -11,14 +11,15 @@ BYPASS_REMEDIATIONS?=false
 
 GOLANGCI_LINT_VERSION = v1.40.1
 BUILD_DIR := build
+PLATFROM?=ocp
 
 .PHONY: all
 all: e2e
 
 .PHONY: e2e
 e2e: ## Run the e2e tests. This requires that the PROFILE and PRODUCT environment variables be set.
 ## idp_fix.patch is used to fix route destination cert for keycloak IdP deployment
-	go test $(TEST_FLAGS) . -profile="$(PROFILE)" -product="$(PRODUCT)" -content-image="$(CONTENT_IMAGE)" -install-operator=$(INSTALL_OPERATOR) -bypass-remediations="$(BYPASS_REMEDIATIONS)" | tee .e2e-test-results.out
+	go test $(TEST_FLAGS) . -platform="$(PLATFROM)" -profile="$(PROFILE)" -product="$(PRODUCT)" -content-image="$(CONTENT_IMAGE)" -install-operator=$(INSTALL_OPERATOR) -bypass-remediations="$(BYPASS_REMEDIATIONS)" | tee .e2e-test-results.out
```

Comment:
I wonder if we can just add in an argument for the assertion file directly, instead of using the platform.

We only need the platform to discover the file through the directory structure in the content, right?